### ISSUE TITLE
Soul Cage - log order is hard to follow

### ISSUE THREAD

**Issue by mplain** [db0/Doomtown-Reloaded-OCTGN#132]
What happened: Soul Cage brings Ivor XP, Ivor brings Junior and Raising Hell, Junior bring another Soul Cage.
What the log shows:

mplain has paid 2 Ghost Rock. 6 is left in their stash
mplain has hired Junior.
:::ERROR::: You need to target a valid card before using this action
You need: ['Dude'].
:::ERROR::: You need to target a valid card before using this action
You need: ['Deed'], or ['Dude'], or ['Outfit'].
mplain has paid 3 Ghost Rock. 3 is left in their stash
Ivor Hawley has purchased Soul Cage.
:> mplain uses Junior to play ['Soul Cage'] from their Deck..
:::ERROR::: You need to target a valid card before using this action
You need: ['Dude', 'Huckster'].
:::ERROR::: You need to target a valid card before using this action
You need: ['Deed'], or ['Dude'], or ['Outfit'].
Ivor Hawley has learned Raising Hell.
:> mplain uses Ivor Hawley to play ['Junior', 'Raising Hell'] from their Boot Hill..
:> mplain boots Soul Cage to play ['Ivor Hawley'] from their Boot Hill..

**Comment by db0** [comment]
That can't be fixed without a ridiculous rework. Just work with what you have :)
